packages/networks/src/index.ts

```diff
@@ -131,13 +131,14 @@ const all: NetworkFromSubstrate[] = [
     website: 'http://laminar.network/'
   },
   {
-    decimals: null,
-    displayName: 'Polymath',
-    network: 'polymath',
+    decimals: [6],
+    displayName: 'Polymesh',
+    genesisHash: ['12fddc9e2128b3fe571e4e5427addcb87fcaf08493867a68dd6ae44b406b39c7'],
```

Comment:
As a first round, would just add the genesisHash which is needed for Ledger-operation.

The rest of the changes fail since this is checked against the ss58-registry on Substrate, so should map, see the crosscheck failure at https://github.com/polkadot-js/common/runs/1697045426?check_suite_focus=true#step:3:36 

(Background to this ... Atm this is duplicated - the idea is that that ss58-rregistry json be split into a seperate repo and be usable from both JS & Rust. At that point the only values here that will remain will be the genesisHashes, i.e. the fields not in the Substrate master edition)

Reply:
Thanks @jacogr - does this mean that changing the network name (`polymath` > `polymesh`) will always fail this crosscheck? If so, I wonder whether it would be possible to push the change through w/o this check?

If that's not possible I'll update to just include the `genesisHash` although ideally we'd also like to change the name at some point.

Reply:
Yes, just remove all the other changes until the ss58 registry on Substrate is updated, e.g. https://github.com/paritytech/substrate/blob/master/ss58-registry.json. (There is a nightly check that picks up discrepancies, so once merged there will apply the changes here)

All the JS stuff ignores it anyway, the important bits it does use is the network and then the genesisHash. (Things like the decimals and token is gets from the chain)

Reply:
Thanks for the details. This PR is updated to just have the genesisHash. I'll create a PR against Substrate/.../ss58-registry.json with the other details.